conf/template/yarp_plugin_device.cpp.in

```diff
@@ -17,9 +17,6 @@
 #  define YARP_PLUGIN_EXPORT YARP_HELPER_DLL_EXPORT
 #endif
 
-using namespace yarp::os;
-using namespace yarp::dev;
```

Comment:
@traversaro That's what happens when examples are not built automatically :grin:
Thanks for noticing, I will fix the examples and add some info to the ChangeLog...